### ISSUE TITLE
`warm-up-command` not working as expected in 0.8.0 fails with `error: No such file or directory (os error 2)`

### ISSUE THREAD

**Issue by kevinjalbert** [rustic-rs/rustic#1209]
I am using rustic version 0.8.0 on MacOS (built via `cargo binstall rustic-rs`), and I am hitting issues with the `warm-up-command` configuration (in my `rustic.toml`).

```
# following https://kmh.prasil.info/posts/rustic-cold-storage-glacier-migration-configuration/#how-to-restore
warm-up-command = "/Users/jalbert/src/shoto/immich_backup/warmup.sh %id"
```

```
[00:00:00] warming up packs...            ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░          0/1
[DEBUG] (1) rustic_core::repository::warm_up: calling ["true ee938082eab1d740f3fac87e1e1cb48d876193a164a41fdecdf09ddf2a2b1548"]...
error: No such file or directory (os error 2)
```

I even tried `warm-up-command = "true %id"` and this didn't work either 🤔.

`warmup.sh` is just a call to `aws s3api restore-object`. When I ran it without `rustic` it started the restoration process correctly.

I believe this is linked to this issue: https://github.com/rustic-rs/rustic/issues/1177, which is merged and released. I'm not sure if my version has that fix though (not sure how to get that change if so -- I'm pretty new to rust/cargo)? Or if there is something else going wrong with my approach here?

**Comment by aawsome** [comment]
@kevinjalbert 
This is one of the few breaking changes in rustic 0.8.0.

You should use 
```
warm-up-command = ["/Users/jalbert/src/shoto/immich_backup/warmup.sh", "%id"]
```
in the config profile file.

**Comment by aawsome** [comment]
For explanation: 

Before 0.8.0, we tried to split the command into args by whitespaces. But whitespaces can also be inside paths, so some extra escaping had to be added..
And this escaping leads to other problems: The TOML format also uses escaping to allow special characters inside a string. Having to deal with those both escaping makes it really messy and and almost impossible to understand what's going on.

So, we decided to allow specifying the warm-up-command as string array and just use it as-is.
